sabaibiometrics/management/commands/create_dummy_data.py

```diff
@@ -1,78 +1,120 @@
+                        diastolic=vitals_number,
+                        temperature=vitals_number,
+                        diabetes_mellitus=["Yes", "No"][visit_number % 2],
+                        heart_rate=vitals_number,
+                        urine_test=["True", "False"][visit_number % 2],
+                        hemocue_count=vitals_number,
+                        blood_glucose=vitals_number,
+                        left_eye_degree=f"+{vitals_number}",
+                        right_eye_degree=f"+{vitals_number}",
+                        left_eye_pinhole=f"+{vitals_number}",
+                        right_eye_pinhole=f"+{vitals_number}",
+                        others=f"others{vitals_number}",
+                    )
+
+                    # Create consult
+                    consult = Consult.objects.create(
+                        visit=visit,
+                        date=timezone.now(),
+                        doctor=doctor,
+                        past_medical_history=f"Medical Diagnosis {vitals_number}",
+                        consultation=f"General Checkup {vitals_number}",
+                        plan=f"Follow-up in {vitals_number} months",
+                        referred_for=f"referred_for_{vitals_number}",
+                        referral_notes=f"referral_notes_{vitals_number}",
+                        remarks=f"remark_{vitals_number}"
+                    )
+
+                    # Create a diagnosis
+                    Diagnosis.objects.create(
+                        consult=consult,
+                        details=f"consult_details{vitals_number}",
+                        category=f"consult_category{vitals_number}",
+                    )
+
+                    # Create medication
+                    medication = Medication.objects.create(
+                        medicine_name=f"Medication {vitals_number}",
+                        quantity=patient_number,
+                        notes=f"Take {vitals_number} as needed"
+                    )
+
+                    # Create medication review
+                    medication_review = MedicationReview.objects.create(
+                        approval=doctor,
+                        quantity_changed=random.randint(-10, 10),
+                        quantity_remaining=medication.quantity + random.randint(-5, 5),
+                        medicine=medication,
+                        date=timezone.now(),
+                        order_status=random.choice(['APPROVED', 'PENDING', 'CANCELLED']),
+                    )
+
+                    # Create an order
+                    Order.objects.create(
+                        consult=consult,
+                        notes=f"order_notes{vitals_number}",
+                        remarks=f"remark_{vitals_number}",
+                        medication_review=medication_review,
+                    )
+
+            self.stdout.write(self.style.SUCCESS("Dummy data created successfully"))
```

Comment:
Any particular reason for using stdout.write() here instead of print?

Reply:
it is easier to see on the terminal and I can find it easily.

Reply:
There can be some unexpected behaviour happening when using stdout.write, so it would be best to use print instead. 

You can try using following if you want to change terminal colour.
``` 
COLOR_RESET = '\033[0m'
def color_hash(s):
    color_codes = [
        '\033[31m',  # Red
        '\033[32m',  # Green
        '\033[33m',  # Yellow
        '\033[34m',  # Blue
        '\033[36m',  # Cyan
        '\033[37m'   # White
    ]
    
    hash_value = hash(s)
    color_index = hash_value % len(color_codes)
    
    return color_codes[color_index]

NAME = input("Name: ")
color = color_hash(NAME)
NAME = color + NAME + COLOR_RESET
print(NAME)
```

Check out ANSI codes in python:
[Print colour text to terminal in python](https://www.studytonight.com/python-howtos/how-to-print-colored-text-in-